example/docker/debian12/Dockerfile

```diff
@@ -5,9 +5,20 @@ RUN echo root:1234 | chpasswd
 
 RUN apt-get update && \
     DEBIAN_FRONTEND=noninteractive apt-get install -y \
-      build-essential binutils lsb-release make git libssl-dev openssh-server wget patchelf && \
+      build-essential binutils lsb-release make git libssl-dev openssh-server wget patchelf software-properties-common flex && \
     rm -rf /var/lib/apt/lists/*
 
+RUN git clone git://gcc.gnu.org/git/gcc.git /gcc && \
+    cd /gcc && \
+    git checkout releases/gcc-13 && \
```

Comment:
PLease use the same version of GCC as ubuntu 2404 is using.

Reply:
+ please use shallow clone and download only a specific release branch. Not the whole repository.